### PR TITLE
additional text for iana registry

### LINKED ISSUE #194: IANA Registry
- Section 4 of RFC 8126 -> supply registration procedures for the registries
- Define where these registries should be placed

### PR DESCRIPTION
**Pull request by c2bo**
Closes #194 

We would register:
CWT Status Mechanisms registry in CWT 
JWT Status Mechanisms registry in JWT 
Status Types registry in OAuth parameters

Any ideas how we could simplify this?

### CODE REVIEW

**Comment by paulbastian** [comment]
looks good to me in general. I would extend this to more than two weeks, if I look back how the current year went by. I think this is a good start for now, so we should merge this first and then evaluate afterwards if we can merge both status registries.

File: draft-ietf-oauth-status-list.md [hunk 1]
```diff
@@ -968,6 +968,19 @@ IANA "JSON Web Token Claims" registry {{IANA.JWT}} established by {{RFC7519}}.
 
 This specification establishes the IANA "Status Mechanism Methods" registry for JWT "status" member values. The registry records the status mechanism method member and a reference to the specification that defines it.
 
+JWT Status Mechanism Methods are registered by Specification Required [RFC5226] after a two-week
```
Comment (c2bo) [inline draft-ietf-oauth-status-list.md:971]:
```suggestion
JWT Status Mechanism Methods are registered by Specification Required [RFC5226] after a three-week
```

File: draft-ietf-oauth-status-list.md [hunk 2]
```diff
@@ -1034,6 +1047,18 @@ IANA "CBOR Web Token (CWT) Claims" registry {{IANA.CWT}} established by {{RFC839
 
 This specification establishes the IANA "Status Mechanism Methods" registry for CWT "status" member values. The registry records the status mechanism method member and a reference to the specification that defines it.
 
+CWT Status Mechanism Methods are registered by Specification Required [RFC5226] after a two-week
```
Comment (c2bo) [inline draft-ietf-oauth-status-list.md:1050]:
```suggestion
CWT Status Mechanism Methods are registered by Specification Required [RFC5226] after a three-week
```

**Comment by c2bo** [comment]
> looks good to me in general. I would extend this to more than two weeks, if I look back how the current year went by. I think this is a good start for now, so we should merge this first and then evaluate afterwards if we can merge both status registries.

JWT/CWT registries use three week periods, the OAuth Parameter one uses two weeks. Should we just align with these registries for each one?

**Review by paulbastian — APPROVED** [review]

**Review by tplooker — APPROVED** [review]

**Pull request merged**